packages/braid-design-system/src/lib/components/private/Popover/Popover.screenshots.tsx

```diff
@@ -0,0 +1,371 @@
+import { type MutableRefObject, type ReactNode, useRef } from 'react';
+import type { ComponentScreenshot } from 'site/types';
+
+import { Box } from '../..';
+import type { BoxProps } from '../../Box/Box';
+import { Placeholder } from '../Placeholder/Placeholder';
+
+import { Popover, type PopoverProps } from './Popover';
+
+const placeholderHeight = 44;
+const triggerWidth = 150;
+const popoverWidth = 200;
+
+const Wrapper = ({
+  children,
+  popoverPlacement = 'bottom',
+  justifyContent = 'center',
+}: {
+  children: (props: {
+    triggerRef: MutableRefObject<HTMLButtonElement | null>;
+  }) => ReactNode;
+  popoverPlacement?: PopoverProps['placement'];
+  justifyContent?: BoxProps['justifyContent'];
+}) => {
+  const triggerRef = useRef<HTMLButtonElement>(null);
+  const wrapperPadding = 'xxlarge';
+
+  return (
+    <Box
+      display="flex"
+      justifyContent={justifyContent}
+      paddingTop={popoverPlacement === 'top' ? wrapperPadding : undefined}
+      paddingBottom={popoverPlacement === 'bottom' ? wrapperPadding : undefined}
+    >
+      <Box ref={triggerRef}>
+        <Placeholder
+          label="Trigger"
+          height={placeholderHeight}
+          width={triggerWidth}
+        />
+      </Box>
+      {children({ triggerRef })}
+    </Box>
+  );
+};
+
+const ScreenshotPopover = ({
+  ...props
+}: Omit<PopoverProps, 'open' | 'lockPlacement' | 'role'>) => (
+  <Popover open lockPlacement role={false} {...props} />
+);
```

Comment:
Same a MenuRenderer screenshots, lets refactor this to a default props object and use Popover directly in the examples